src/main/java/ch/njol/skript/effects/EffVectorRotateAroundAnother.java

```diff
@@ -63,18 +60,18 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean kleenean
 
 	@SuppressWarnings("null")
 	@Override
-	protected void execute(Event e) {
-		Vector v2 = second.getSingle(e);
-		Number d = degree.getSingle(e);
+	protected void execute(Event event) {
+		Vector v2 = second.getSingle(event);
+		Number d = degree.getSingle(event);
 		if (v2 == null || d == null)
 			return;
-		for (Vector v1 : first.getArray(e))
+		for (Vector v1 : first.getArray(event))
 			VectorMath.rot(v1, v2, d.doubleValue());
```

Comment:
Do you think the budget would stretch to a few extra letters in our variable names?

Reply:
I think we might be able to fit in a few more if we hide it under facilities cost.